maven-model-builder/src/main/java/org/apache/maven/model/building/DefaultModelBuilder.java

```diff
@@ -1352,6 +1449,19 @@ private void importDependencyManagement( Model model, ModelBuildingRequest reque
         return null;
     }
 
+    private <T> T getCache( ModelCache modelCache, Source source, ModelCacheTag<T> tag )
```

Comment:
The method name `getCache` suggests that it returns a cache. But one of its parameters is _named_ "cache" and it returns a `<T>` instance. Does the name `getCache` match with what the method does?

Reply:
Somebody else introduced `getCache()` and `putCache()` based on GAV. My guess: he had to choose between `get()`/`put()` and `getFromCache()`/`putIntoCache()` and chose a little of both( poldermodel ;) ), resulting in probably the worst, especially for `getChache()`.
I went for consistency, but since these are private methods we should be safe to rename them.